pkg/statistics/hot_cache.go

```diff
@@ -80,7 +80,7 @@ func (w *HotCache) CheckReadAsync(task func(cache *HotPeerCache)) bool {
 func (w *HotCache) RegionStats(kind utils.RWType, minHotDegree int) map[uint64][]*HotPeerStat {
 	ret := make(chan map[uint64][]*HotPeerStat, 1)
 	collectRegionStatsTask := func(cache *HotPeerCache) {
-		ret <- cache.RegionStats(minHotDegree)
+		ret <- cache.PeerStats(minHotDegree)
```

Comment:
Why change the name, it's not consistent with other places.

Reply:
Okay, I'll keep the TODO and change them in another PR uniform.